images/Dockerfile.py3.aws

```diff
@@ -14,9 +14,6 @@ RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python get-pip.py
 RUN python3.8 -m pip install \
     filelock \
     fire \
-    google-api-python-client \
```

Comment:
I think this is unnecessary to have those dependencies in optional-test-infra worker image.